src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
```

Comment:
```suggestion
public class Ball {
```

Ball should not extend Lottery, as these classes does not have common state or behavior